dashboard/src/components/api/devfile-registry.factory.ts

```diff
@@ -86,4 +108,26 @@ export class DevfileRegistry {
     } catch (e) {
     }
   }
+
+   /**
+   * Returns memory limit.
+   * @param {string} memoryLimit
+   * @returns {number}
+   */
+  getMemoryLimit(memoryLimit: string): number {
+    if (!memoryLimit) {
+      return -1;
+    }
+    const regExpExecArray = /^([0-9]+)([a-zA-Z]{1,3})$/.exec(memoryLimit);
```

Comment:
It would be nice to improve regexp using values from enum MemoryUnit

Reply:
I check the MemoryUnit in this place of codehttps://github.com/eclipse/che/pull/14270/files/35a5ea6771cc28eee7506acaf132e17cc93bf51a#diff-6296915bbb52eb6ee7111f8e2fd1e2feR126
In this case, it will be simpler to change something - just change the units enum in the head of the file